app/models/cinstance.rb

```diff
@@ -111,17 +111,20 @@ def validate_plan_is_unique?
   validate :same_service, on: :update, if: :plan_id_changed?
 
   # letter, number, underscore (_), hyphen-minus (-), dot (.), base64 format
-  # In base64 encoding, the character set is [A-Z,a-z,0-9,and + /], if rest length is less than 4, fill of '=' character.
-  # ^([A-Za-z0-9+/]{4})* means the String start with 0 time or more base64 group.
-  # ([A-Za-z0-9+/]{4}|[A-Za-z0-9+/]{3}=|[A-Za-z0-9+/]{2}==) means the String must end of 3 forms in [A-Za-z0-9+/]{4} or [A-Za-z0-9+/]{3}= or [A-Za-z0-9+/]{2}==
+  # In base64 encoding, the character set is [A-Z,a-z,0-9,and +], if rest length is less than 4, fill of '=' character.
+  # ^([A-Za-z0-9+]{4})* means the String start with 0 time or more base64 group.
+  # ([A-Za-z0-9+]{4}|[A-Za-z0-9+]{3}=|[A-Za-z0-9+]{2}==) means the String must end of 3 forms in [A-Za-z0-9+]{4} or [A-Za-z0-9+]{3}= or [A-Za-z0-9+]{2}==
   # matches also the non 64B case with (\A[\w\-\.]+\Z)
-  USER_KEY_FORMAT = /(([\w\-\.]+)|([A-Za-z0-9+\/]{4})*([A-Za-z0-9+\/]{4}|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{2}==))/
-
-  validates :application_id, format: { with: /\A[\x20-\x7E]+\Z/ }, length: { in: 4..255 }
+  # NOTE: base64 format also accepts forward slash (/), however we don't allow it because of the restriction on the backend
+  USER_KEY_FORMAT = /(([\w\-.]+)|([A-Za-z0-9+]{4})*([A-Za-z0-9+]{4}|[A-Za-z0-9+]{3}=|[A-Za-z0-9+]{2}==))/.freeze
```

Comment:
Yeah, that's a drawback... But considering that such services wouldn't work now anyway (because the key would't be created on backend), I think that's fine.

And also, we may later fix the issue on the backend instead, and bring back slashes in system.